.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,16 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
+    <inspection_tool class="PyUnresolvedReferencesInspection" enabled="true" level="WARNING" enabled_by_default="true">
+      <option name="ignoredIdentifiers">
+        <list>
+          <option value="sarra.sr_cache.check.mock_calls" />
```

Comment:
value="sarra.sr_cache.check.mock_calls"

I am not sure your suppose to have that kind of config here.